docs: Update docs according to the style guide (#6512)
cube-js/cube: docs/content/Examples-Tutorials-Recipes/Recipes/Query-acceleration/Refreshing-select-partitions.mdx

In our case, each order has `created_at` and `updated_at` properties. The
`updated_at` property is the last order update timestamp. To create a
pre-aggregation with partitions, we need to specify the
[`partition_granularity` property](https://cube.dev/docs/schema/reference/pre-aggregations#partition-granularity).
Partitions will be split monthly by the `created_at` dimension.

<CodeTabs>

```yaml
cubes:

>   - name: orders
>     pre_aggregations:
>       - name: orders
>         type: rollup
>         dimensions:
>           - number
>           - status
>           - created_at
>           - updated_at
>         time_dimension: created_at
>         granularity: day
>         partition_granularity: month # this is where we specify the partition
>         refreshKey:
>           sql: SELECT max(updated_at) FROM public.orders # check for updates of the updated_at property

Author: ```suggestion
  - name: orders
    # ...
    
    pre_aggregations:
      - name: orders
        type: rollup
        dimensions:
          - number
          - status
          - created_at
          - updated_at
        time_dimension: created_at
        granularity: day
        partition_granularity: month # this is where we specify the partition
        refresh_key:
          sql: SELECT max(updated_at) FROM public.orders # check for updates of the updated_at property
```

```

```javascript

> cube("orders", {

Author: ```suggestion
cube(`orders`, {
  // ...
  
```

Note that the query for two partitions is the same. It's the reason why **all
partitions** will be updated.

How do we fix this and update only the partition for August? We can use the
[`FILTER_PARAMS`](https://cube.dev/docs/schema/reference/cube#filter-params) for
that!

Let's update our pre-aggregation definition:

<CodeTabs>

```yaml
cubes:

>   - name: orders

Author: ```suggestion
  - name: orders
    # ...
    
```

    pre_aggregations:
      - name: orders
        type: rollup
        dimensions:
          - number
          - status
          - created_at
          - updated_at
        time_dimension: created_at
        granularity: day
        partition_granularity: month # this is where we specify the partition
        refreshKey:
          sql: "SELECT max(updated_at) FROM public.orders WHERE {FILTER_PARAMS.orders.created_at.filter('created_at')}"
```

```javascript

> cube("orders", {

Author: ```suggestion
cube(`orders`, {
  // ...
  
```

  pre_aggregations: {
    orders: {
      type: `rollup`,
      dimensions: [number, status, created_at, updated_at],
      time_dimension: created_at,
      granularity: `day`,
      partition_granularity: `month`,
      refresh_key: {

>         sql: `SELECT max(updated_at) FROM public.orders WHERE ${FILTER_PARAMS.orders.created_at.filter('created_at')}`

Author: ```suggestion
        sql: `
          SELECT MAX(updated_at)
          FROM public.orders
          WHERE ${FILTER_PARAMS.orders.created_at.filter('created_at')}
        `
```
